kolibri/plugins/audio_mp3_render/assets/src/module.js

```diff
@@ -1,17 +1,13 @@
 
-// This is aliased for your convenience!
-const KolibriModule = require('kolibri_module');
+const ContentRendererModule = require('content_renderer_module');
+const AudioComponent = require('./vue/index');
 
-class AudioMP3Module extends KolibriModule {
-  render(contentData, renderArea) {
-    const audioTag = document.createElement('audio');
-    audioTag.autoplay = true;
-    audioTag.src = 'https://freemusicarchive.org/music/download/8a46d5ff69d7c426d2a0a6854d96a683dec17802';
-    audioTag.controls = true;
-    const titleTag = document.createElement('h1');
-    titleTag.innerHTML = contentData.id;
-    renderArea.appendChild(titleTag);
-    renderArea.appendChild(audioTag);
+class AudioMP3Module extends ContentRendererModule {
+  get rendererComponent() {
```

Comment:
Content renderers now just need to define a getter to return the component, and the content type that they handle.